examples/imitation_learning_openbot/generate_dataset.py

```diff
@@ -122,65 +107,43 @@
 
-        # initialize the driving policy with the desired trajectory 
-        policy.reset(obs, env_step_info)
+        # initialize the driving policy with the desired trajectory
+        policy.reset(obs, trajectory)
 
         if args.benchmark:
             start_time_seconds = time.time()
         else:     
             # build the episode data folder and its subfolders following the guidelines of the OpenBot public repository 
             # https://github.com/isl-org/OpenBot/tree/master/policy#data-collection
-            scene_dir = os.path.realpath(os.path.join(split_dir, episode["scene_id"]))
+            scene_dir   = os.path.realpath(os.path.join(split_dir, episode["scene_id"]))
             episode_dir = os.path.realpath(os.path.join(scene_dir, "%04d" % episode["index"]))
-            image_dir = os.path.realpath(os.path.join(episode_dir, "images"))
-            sensor_dir = os.path.realpath(os.path.join(episode_dir, "sensor_data"))
-            plots_dir = os.path.realpath(os.path.join(episode_dir, "plots"))
+            image_dir   = os.path.realpath(os.path.join(episode_dir, "images"))
+            sensor_dir  = os.path.realpath(os.path.join(episode_dir, "sensor_data"))
+            plots_dir   = os.path.realpath(os.path.join(episode_dir, "plots"))
             os.makedirs(image_dir, exist_ok=True)
             os.makedirs(sensor_dir, exist_ok=True)
             os.makedirs(plots_dir, exist_ok=True)
 
-            control_data  = np.empty([args.num_iterations_per_episode, 2], dtype=np.float32) # control_data observations made by the agent during a episode
-            state_data    = np.empty([args.num_iterations_per_episode, 6], dtype=np.float32) # state_data observations made by the agent during an episode
-            waypoint_data = np.empty([args.num_iterations_per_episode, 3], dtype=np.float32) # waypoints being tracked by the agent during an episode
-            compass_data  = np.empty([args.num_iterations_per_episode, 3], dtype=np.float32) # compass observations made by the agent during a episode
+            control_data  = np.empty([args.num_iterations_per_episode, 2], dtype=np.float64) # control_data observations made by the agent during a episode
+            state_data    = np.empty([args.num_iterations_per_episode, 6], dtype=np.float64) # state_data observations made by the agent during an episode
+            waypoint_data = np.empty([args.num_iterations_per_episode, 3], dtype=np.float64) # waypoints being tracked by the agent during an episode
+            compass_data  = np.empty([args.num_iterations_per_episode, 3], dtype=np.float64) # compass observations made by the agent during a episode
             time_data     = np.empty([args.num_iterations_per_episode], dtype=np.int64)      # time stamps of observations made by the agent during an episode
             frame_data    = np.empty([args.num_iterations_per_episode], dtype=np.int32)      # frame ids
 
```

Comment:
`num_iterations_executed`

Reply:
right, this is a bug, i.e., multiplying the `timestamp` value by `10000` will not give us time in `ns`. Updated the name to just `timestamp` as opposed to `timestamp[ns]`.